Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/brassknuckles.yml

```diff
@@ -0,0 +1,27 @@
+- type: entity
+  name: Brass knuckles
+  parent: ClothingHandsGlovesBoxingRed
+  id: WeaponBrassKnuckles
+  description: Ever wanted to punch someone in the mouth harder then regular? Well S.E.S.W.C has what you need, take your anger out with fashion on your enemies
+  components:
+  - type: Sprite
+    sprite: _NF/Objects/Weapons/Melee/brassknuckles.rsi
+    state: icon
+  - type: Clothing
+    sprite: _NF/Objects/Weapons/Melee/brassknuckles.rsi
+  - type: MeleeWeapon
+    autoAttack: true
+    attackRate: 2
+    damage:
+      types:
+       Blunt: 10
+    soundHit:
+      collection: Punch
+    animation: WeaponArcFist
+    mustBeEquippedToUse: true
+  - type: Fiber
+    fiberMaterial: fibers-durathread
+    fiberColor: fibers-yellow
+  - type: FingerprintMask
```

Comment:
How do brass knuckles mask your fingerprints?  Why do they leave yellow fibers?

Reply:
Yellow is the closes and if i didnt it would have pull from the parent

Reply:
Again, how do brass knuckles mask your fingerprints?

Reply:
I agree that they shouldn't leave fibers

Reply:
how are you wanting me to prevent the pull from the parent